pina/graph.py

```diff
@@ -162,6 +165,18 @@ def _preprocess_edge_index(edge_index, undirected):
             edge_index = to_undirected(edge_index)
         return edge_index
 
+    def extract(self, labels):
```

Comment:
maybe a better option would be `def extract(self, labels, attr='x')` where attr is where you want to extract the label from.